src/services/moderation.service.ts

```diff
@@ -385,9 +389,9 @@ export class ModService {
   private async _kickUser(member: GuildMember) {
     await member.send(
       'You are being kicked for too many warnings\n' +
-        `You currently have been warned ${this._KICK_THRESH} times.\n` +
-        `After ${this._SUSPEND_THRESH} warnings, you will have restricted access to the server.\n` +
-        `After ${this._BAN_THRESH} warnings, you will be banned permanently.`
+      `You currently have been warned ${this._KICK_THRESH} times.\n` +
+      `After ${this._SUSPEND_THRESH} warnings, you will have restricted access to the server.\n` +
+      `After ${this._BAN_THRESH} warnings, you will be banned permanently.`
```

Comment:
There's quite of few unrelated formatting changes in this. The prettier vsc plugin has all of our styling if you don't have that installed. Or just try to remove these changes